corehq/apps/reports/models.py

```diff
@@ -467,6 +467,13 @@ def get_report_content(self, lang, attach_excel=False):
                 file_obj = dispatch_func(render_as='excel')
             else:
                 file_obj = None
+            if response.status_code == 302:
+                return _("We are sorry, but your saved report '%(config_name)s' "
+                         "is no longer accessible because the owner %(username)s "
+                         "is no longer active.") % {
+                    'config_name': self.name,
+                    'username': self.owner.username
+                }, None
```

Comment:
but ... the code is pep8 ... that means it's good code right? :stuck_out_tongue_winking_eye: